lib/ewasm/system/transform-i64.wast

```diff
@@ -0,0 +1,13 @@
+(module
```

Comment:
You could use `wabt` (as an npm module) as part of the build system to generate the `wasm` of this.

Reply:
Did you mean this [wabt.js](https://github.com/AssemblyScript/wabt.js)? Seems a bit buggy, haven't been able to get it to work yet. Will keep working on it.

Or rather including the wabt toolkit itself?

Reply:
Meant wabt.js